MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -1094,6 +1117,9 @@
         }
     }
 
+    static const uint32_t kAMDVendorId = 0x1002;
+    _metalFeatures.subgroupSize = (_properties.vendorID == kAMDVendorId) ? 64 : 32;
+
```

Comment:
It's possible to fetch the `_metalFeatures` struct from user code using `VK_MVK_moltenvk`. We probably shouldn't set this unconditionally... or really, at all on iOS or tvOS until we figure out what to do there.